tests/travis.sh

```diff
@@ -2,7 +2,5 @@
 
 set -ev
 
-# Run all debug tests
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
I am not sure what the code is doing here. Please explain a little bit.

Reply:
This is to run the test suite every time the code is built on travis. This is because currently we do not, and so the test coverage thinkgs we're covering 0% of the code